packages/core/src/types/AffineMatrix.ts

```diff
@@ -0,0 +1,8 @@
+type AffineMatrix = [
```

Comment:
Do you really need to define yet another matrix format?  How about affineMatrix as a function producing a gl-matrix mat4 instance instead, taking four arrays like this as the arguments, and producing the standard mat4 representation.

Reply:
As discussed in the meeting, I will do it later